rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1339,3 +1342,46 @@ def test_old_versions_raise_error(user_data_dir):  # pylint: disable=unused-argu
             msg_aggregator=msg_aggregator,
         )
     assert 'Your account was last opened by a very old version of rotki' in str(upgrade_exception)
+
+
+def test_ongoing_upgrades(user_data_dir):
```

Comment:
Call them unfinished upgrades. As ongoing is a misleading title. It's not ongoing. It started, but was stopped in the middle.

```suggestion
def test_unfinished_upgrades(user_data_dir):
```